go.mod

```diff
@@ -231,7 +231,7 @@ require (
 	golang.org/x/exp/typeparams v0.0.0-20240314144324-c7f7c6466f7f // indirect
 	golang.org/x/mod v0.17.0 // indirect
 	golang.org/x/net v0.25.0 // indirect
-	golang.org/x/oauth2 v0.21.0 // indirect
+	golang.org/x/oauth2 v0.23.0 // indirect
```

Comment:
Why this dependency has changed?

Reply:
@dbozzato81, I didn't do anything except the change at line 9 and "go mod tidy".